climada/hazard/tc_tracks.py

```diff
@@ -625,19 +671,24 @@ def _read_file_emanuel(self, path, hemisphere='S', rmw_corr=False):
         path : str
             absolute path of file to read.
         hemisphere : str, optional
-            'S', 'N' or 'both'. Default: 'S'
+            For global data sets, restrict to northern ('N') or southern ('S') hemisphere.
+            Use 'both' to disable this restriction. Default: 'S'
         rmw_corr : str, optional
             If True, multiply the radius of maximum wind by factor 2. Default: False.
         """
-        if hemisphere == 'S':
-            hem_min, hem_max = -90, 0
-        elif hemisphere == 'N':
-            hem_min, hem_max = 0, 90
-        else:
-            hem_min, hem_max = -90, 90
-
         LOGGER.info('Reading %s.', path)
         data_mat = matlab.loadmat(path)
+        basin = str(data_mat['bas'][0])
+
+        hem_min, hem_max = -90, 90
+        if basin == "GB":
+            if hemisphere == 'S':
+                hem_min, hem_max = -90, 0
+                basin = "S"
+            elif hemisphere == 'N':
+                hem_min, hem_max = 0, 90
+                basin = "N"
+
```

Comment:
This makes sure that the basin information contained in the data set is used if available.